cvat/apps/engine/admin.py

```diff
@@ -84,8 +84,20 @@ class TaskAdmin(admin.ModelAdmin):
     def has_add_permission(self, request):
         return False
 
+class CloudStorageAdmin(admin.ModelAdmin):
+    date_hierarchy = 'updated_date'
```

Comment:
Probably you want to add here display_name as well.

Reply:
Fixed